scripts/get-fonts.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
 set -e
 
 FONTDIR="./fonts"
```

Comment:
Nice work @mapmeld, would it be possible to add something like this so that the installation destination could be customized?
```suggestion
FONTDIR="${FONTDIR:-./fonts}"
```

Reply:
make sense ✔️